src/components/MakeComment/MakeComment.jsx

```diff
@@ -0,0 +1,95 @@
+import { useEffect, useState } from "react";
+import "./MakeComment.css";
+import { postComment } from "../../utils/app";
+import { useParams } from "react-router-dom";
+import CommentCard from "../CommentCard/CommentCard";
+
+const MakeComment = () => {
+
+  const params = useParams();
+  const article_id = params.article_id;
+
+  const [makeCommentState, setMakeCommentState] = useState(false);
+  const [currentUser, setCurrentUser] = useState("jessjelly");
+  const [commentBody, setCommentBody] = useState("");
+  const [commentComplete, setCommentComplete] = useState(false);
+  const [addedComment, setAddedComment] = useState("");
+
+ 
+  const makeComment = () => {
+    setMakeCommentState(true);
+  };
+
+ 
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    setCommentComplete(true);
+    postNewComment(article_id, currentUser, commentBody);
+  };
+
+  const postNewComment = (article_id, userName, commentBody) => {
+    const newComment = {
+      article_id: article_id,
+      userName: userName,
+      body: commentBody,
+    };
+
+    postComment(newComment).then((response) => {
+      setAddedComment(response);
+      setCommentBody("");
+    });
+  };
+
+  if (commentComplete) {
+    return (
+      <>
+        <section className="makeCommentContainer">
+          <h3> Your comment has been added. Thank you</h3>
+        </section>
+        <section>
+          <CommentCard
+            className="commentContainer"
+            key={addedComment.comment_id}
+            comment={addedComment}
+          />
+        </section>
+      </>
+    );
+  }
+
+  if (!makeCommentState) {
+    return (
+      <section
+        className="makeCommentContainer"
+        onClick={() => {
+          makeComment();
+        }}
+      >
+        <h3> Add A Comment</h3>
```

Comment:
i'd make this a button with an onClick - you can style this how you like (i.e. if you dont want it to look like a button) but the section tag is acting as a button so this can be confusing in terms of accessibility!